src/pages/AdminTemplates/AdminTemplates.scss

```diff
@@ -0,0 +1,169 @@
+@import "scss/constants.scss";
+
+.templates-page {
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 5rem;
+
+  &__nav-buttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 80%;
+
+    &--cancel {
+      padding: 0.5rem 1.5rem;
+      border-radius: 22px;
+      background-color: lighten($color: $saturated-black, $amount: 10);
+      color: $white;
+      border: none;
+
+      &:hover {
+        background-color: lighten($color: $saturated-black, $amount: 20);
+      }
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    &--confirm {
+      padding: 0.5rem 1.5rem;
+      border-radius: 22px;
+      background-color: $primary;
+      color: $white;
+      border: none;
+
+      &:hover {
+        background-color: lighten($color: $primary, $amount: 10);
+      }
+
+      &:focus {
+        outline: none;
+      }
+    }
+  }
+
+  &__template-preview {
+    width: 60%;
+    height: 60vh;
```

Comment:
Why are we using `60vh` here, rather than a `%`/etc? In most cases if not all cases we want to stick to `%`.